cfgov/v1/tests/test_author_names.py

```diff
@@ -0,0 +1,25 @@
+from django.test import TestCase
+from v1.models.base import CFGOVPage
+
+class TestAuthorNames(TestCase):
+
+    def test_names_get_sorted_alphabetically(self):
+        page = CFGOVPage()
+        page.authors.add('Ross Karchner','Richa Agarwal', 'Andy Chosak', 'Will Barton')
+        expected_result = ['Richa Agarwal', 'Will Barton', 'Andy Chosak', 'Ross Karchner']
+        self.assertEquals(page.get_authors(), expected_result)
+    
+    def test_no_authors(self):
+        page = CFGOVPage()
+        self.assertEquals(page.alphabetize_authors(), [])
```

Comment:
@vickumar1981 Yeah, I went back and forth on this a few times.  I wanted a function to reflect that things do in fact get alphabetized when calling `get_authors()`, but otherwise the unit tests really are for `alphabetize_authors()` which has the logic in it.  Perhaps it will make more sense to have a test (in addition to these) that simply asserts `alphabetize_authors()` is called when `get_authors()` is?